static/css/theme.css

```diff
@@ -642,7 +642,7 @@ code + .copy-to-clipboard {
 pre {
     padding: 1rem;
     margin: 2rem 0;
-    background: #1d1f21;
+    background: #282c34;
```

Comment:
Where are these new values coming from?

Reply:
One makes the copy button visible by making it lighter, and the other makes the border color match the new background color of the highlight.js theme.
I quickly edited old template:
![hljs](https://user-images.githubusercontent.com/13084362/54625832-4ff8aa80-4a70-11e9-9d6f-dd8af8538131.PNG)